themes/custom-theme/theme/base/login/messages/messages_en.properties

```diff
@@ -69,13 +71,13 @@ termsTitle=Terms and Conditions
 termsText=<p>Terms and conditions to be defined</p>
 termsPlainText=Terms and conditions to be defined.
 termsAcceptanceRequired=You must agree to our terms and conditions.
-acceptTerms=I agree to the terms and conditions
+acceptTerms=By signing up, <a href="https://katadocs.notion.site/General-Terms-and-Conditions-Accommodation-Suppliers-seller-355d0b04ef7c4d2884be1042e96066a8" target="_blank">I accept the Terms and Conditions</a>
 
 recaptchaFailed=Invalid Recaptcha
 recaptchaNotConfigured=Recaptcha is required, but not configured
 consentDenied=Consent denied.
 
-noAccount=Don't have an account?
+noAccount=Don`t have an account?
```

Comment:
```suggestion
noAccount=Don't have an account?
```

Reply:
will update with a double `'` to have it escaped